### ISSUE TITLE
A11y_.NETCore3.0(WPF)_WPFExpenseItStandAlone_ScreenReader: Narrator does not read the tooltip information for the list of employees.

### ISSUE THREAD

**Issue by grubioe** [dotnet/wpf#1463]
Copied from VSO [951726](https://devdiv.visualstudio.com/DevDiv/_queries/edit/951726/)

Check out Accessibility Insights! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: .NET Core 3.0 (WPF)

.NET Core SDK version:  3.0.100-preview8-013442
Windows Version: Windows10

ScreenReader Version: Narrator

Pre-requisite:

1) Download the latest .NET Core 3 daily build by going to this link.
2) Install Windows (x86 or x64) versions of sdk listed under Master (3.0.x Runtime) by clicking "Installer" link. 

Repro Steps:

1) Run exe file for "ExpenseItDemo" from the build app folder provided.
2) Main Window for "ExpenseItDemo" solution will appear. ​
3) Start Narrator and move to list of employees under set of Employees radiobuttons.

Actual:

Narrator does not read the tooltip conveying instruction "Choose Employee Name" along with list of employees.

This issue has been observed only with Narrator and not with NVDA.

Expected:

Narrator should read the tooltip conveying instruction "Choose Employee Name" along with list of employees.

User Impact:

ScreenReader users will not get instruction given in the tooltip which is necessary for creating Expense Report. 

Recommendations:

Please refer to below URL which is repository for accessibility bug fixes.

https://microsoft.sharepoint.com/teams/msenable/Pages/Home.aspx

MAS Reference:

MAS 1.3.1

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Comment by arpitmathur** [comment]
This is a narrator bug. NVDA reads the "choose employee name" tooltip before reading out the options.
